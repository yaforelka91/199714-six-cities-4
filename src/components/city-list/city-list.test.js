import React from 'react';
import renderer from 'react-test-renderer';
import CityList from './city-list.jsx';

const cities = [
  {
    id: 1,
    name: `city 1`,
    coords: [0, 0],
  },
  {
    id: 2,
    name: `city 2`,
    coords: [0, 0],
  }
];

describe(`CityListSnapshot`, () => {
  it(`should render CityList`, () => {
    const tree = renderer.create(
        <CityList
          citiesList={cities}
          activeCity={cities[0]}
          onCityNameClick={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
