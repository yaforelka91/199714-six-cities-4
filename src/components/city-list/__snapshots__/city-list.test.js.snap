// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityListSnapshot should render CityList 1`] = `
<ul
  className="locations__list tabs__list"
>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
      onClick={[Function]}
    >
      <span>
        city 1
      </span>
    </a>
  </li>
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
      onClick={[Function]}
    >
      <span>
        city 2
      </span>
    </a>
  </li>
</ul>
`;
